lib/UR/DataSource/MySQL.pm

```diff
@@ -39,16 +39,7 @@ sub _format_sql_like_escape_string { return }
 
 sub can_savepoint { 1;} 
 
-sub does_support_limit_offset {
-    my($self, $bx) = @_;
-
-    my $tmpl = $bx->template;
-    if ($tmpl->offset and !defined($tmpl->limit)) {
-        return 0;  # Can't have offset without limit
-    } else {
-        return 1;
-    }
-}
+sub does_support_limit_offset { 0 }
```

Comment:
If you changed https://github.com/genome/UR/blob/3573d2f1e49d3b20b1a393cb2d005a2f42eebd5d/lib/UR/DataSource/RDBMS.pm#L1601 to pass in `$rule`, does that mean this commit isn't necessary?

The other place `does_support_limit_offset()` is called already passes it in: https://github.com/genome/UR/blob/3573d2f1e49d3b20b1a393cb2d005a2f42eebd5d/lib/UR/Context/LoadingIterator.pm#L76